docs/conf.py

```diff
@@ -107,6 +107,14 @@ def setup(app: Sphinx) -> None:
     app.connect("autodoc-skip-member", skip)
 
 
+ignored_modules = [
+    "async_rediscache",
+]
+
+nitpick_ignore_regex = [
+    ("py:.*", "|".join([f".*{entry}.*" for entry in ignored_modules])),
+]
```

Comment:
This is needed since async_rediscache does not have an objects.inv file available for autodocs

Reply:
I'll add a comment to make it clear what it's doing.

Reply:
https://github.com/python-discord/bot-core/pull/42/commits/08d4b717f1d5b6c66137a3b0d333799f024b86df